### PR TITLE
Allows admin to send user feedback on reject data access

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)

- When admin clicks reject, popup with text input is opened for the admin to enter reason of rejecting request.

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?

- git pull origin staging
- git checkout ft-data-access-feedback
- npm run stage
- Visit the candidates' table, its best you reject your own past request which has your email so you are able to receive the feedback in your mailbox

#### What are the relevant tickets?

- [AN-234](https://airqoteam.atlassian.net/browse/AN-234?atlOrigin=eyJpIjoiNTZjYTk0OGQ0YTZiNGVlZTk0ZDRhODEwZWNiMzU3MmMiLCJwIjoiaiJ9)

#### Screenshots
feedback

feedback

### CODE REVIEW

File: netmanager/src/views/pages/CandidateList/components/CandidatesTable/CandidatesTable.js [hunk 1]
```diff
@@ -183,6 +191,34 @@ const CandidatesTable = (props) => {
...
+        return dispatch(
+          updateMainAlert({
+            show: true,
+            message: err.response.data.message,
+            severity: 'error'
+          })
+        );
+      });
+  };
+
```
Comment (Codebmk) [inline netmanager/src/views/pages/CandidateList/components/CandidatesTable/CandidatesTable.js:221]: Changes: Sends feedback and calls modify candidate

File: netmanager/src/views/pages/CandidateList/components/CandidatesTable/CandidatesTable.js [hunk 2]
```diff
@@ -404,6 +451,41 @@ const CandidatesTable = (props) => {
...
+            onClick={() => {
+              setOpenMessagePopup(false);
+              setOpenDel(true);
+            }}
+            color="primary"
+          >
+            Finish
+          </Button>
+        </DialogActions>
+      </Dialog>
```
Comment (Codebmk) [inline netmanager/src/views/pages/CandidateList/components/CandidatesTable/CandidatesTable.js:488]: Changes: Dialog with Feedback message input field

File: netmanager/src/views/pages/CandidateList/components/CandidatesTable/CandidatesTable.js (hunk 1 above)
Comment (Baalmart) [reply netmanager/src/views/pages/CandidateList/components/CandidatesTable/CandidatesTable.js:221]: Alright, thanks @Codebmk , great stuff!

**Review by Baalmart — APPROVED** [review]
Thanks @Codebmk , this is great!

In another PR perhaps, could we consider adding more enhancements? More like: request for a meeting with the person requesting for data access?

Thanks!

**Pull request merged**